tensorboard/plugins/mesh_visualizer/mesh_plugin.py

```diff
@@ -0,0 +1,253 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""TensorBoard 3D mesh visualizer plugin."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import collections
+import numpy as np
```

Comment:
Please space import sections appropriately:

```python
# Parser directives
from __future__ import absolute_import
from __future__ import division
from __future__ import print_function

# Standard library modules
import collections

# Third-party modules
import numpy as np
import six
from werkzeug import wrappers

# First-party modules
from tensorboard.backend import http_util
from tensorboard.plugins import base_plugin
from tensorboard.plugins.mesh_visualizer import metadata
from tensorboard.plugins.mesh_visualizer import plugin_data_pb2
from tensorboard.util import tensor_util
```

Each section is sorted by the module path.

Reply:
Sorry, to clarify: the “`# Standard library modules`”, etc. comments
were just for your benefit and shouldn’t literally be in the code.